nixnet/database/_ecu.py

```diff
@@ -37,52 +40,113 @@ def __repr__(self):
         return '{}(handle={})'.format(type(self).__name__, self._handle)
 
     @property
-    def clst_ref(self):
-        return _props.get_ecu_clst_ref(self._handle)
+    def cluster(self):
+        # type: () -> _cluster.Cluster
+        """:any:`Cluster`: Returns the parent cluster to which the ECU is connected.
```

Comment:
So it was decided to support back references?

I assumed back references would do the delayed import because I'm assuming that is a less common case.  Why is it that the child references are doing the delayed import?

Reply:
I was unaware that supporting back references was ever in question. I believe they've been there since you added a bunch of properties back in #52. If there is an issue here, please fill me in on details.

I was initially trying to keep most of the delayed imports in Pdu since I heard from Jeff L that they aren't heavily used. After thinking about it more, I like limiting delayed imports to back references. I can also keep Frame, Signal, and SubFrame from having to import Pdu all the time, which is actually what we want if Pdus don't in fact see much usage.

Reply:
@d-bohls 
> I was unaware that supporting back references was ever in question.

I first brought up the import issue in https://github.com/ni/nixnet-python/issues/48
> 74 mostly resolves this for System. The problem is circular imports. For System, I just removed the one place that would require circular imports. For Database, I held off until we decide how to resolve it.

and then just comment that it needs to be addressed in https://github.com/ni/nixnet-python/issues/211.
> - All other ref properties returning objects. A lot of these have circular imports which
will need to be addressed.
> - Exposing descendant find (like myFrameA.mySignal). Via getitem, you can get a direct
descendant by name but not indirect descendants. Each parent object
will need an explicit find added to it. This will also run into
problems with circular imports.

Reply:
I'm still confused. Back references were already in there (since #52), so already supported as I understand. I'm just changing them to return objects instead of refs.

Reply:
>  The following support is included:
> - CAN and LIN protocol
> - Frames, Signals, and frame/signal conversion
> - Database import
> - For a complete list of supported features and functions, see the documentation.

From https://github.com/ni/nixnet-python

What got implemented in #52 should not be viewed as "supported" or even usable.  I did a quick and dirty implementation of every database property. For example, ref properties were unusable because they just returned integers.  

My comments in other issues were to say that there is still design work to be done.